<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvme</class>
 <widget class="QMainWindow" name="mvme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionOpen_new"/>
    <addaction name="separator"/>
    <addaction name="actionSave_histogram"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout_mvme"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Display">
   <property name="text">
    <string>New Display</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_new">
   <property name="text">
    <string>Open new</string>
   </property>
  </action>
  <action name="actionSave_histogram">
   <property name="text">
    <string>Save histogram</string>
   </property>
  </action>
  <action name="actionAbout_mvme">
   <property name="text">
    <string>About mvme</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>mvme</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_mvme</sender>
   <signal>activated()</signal>
   <receiver>mvme</receiver>
   <slot>displayAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_new</sender>
   <signal>activated()</signal>
   <receiver>mvme</receiver>
   <slot>createChild()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCascade</sender>
   <signal>activated()</signal>
   <receiver>mvme</receiver>
   <slot>cascade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTile</sender>
   <signal>activated()</signal>
   <receiver>mvme</receiver>
   <slot>tile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayAbout()</slot>
  <slot>createChild()</slot>
  <slot>tile()</slot>
  <slot>cascade()</slot>
 </slots>
</ui>
